uint8 motor_id
string item_name
---
bool comm_result
uint32 value
